.gitignore

```diff
@@ -140,4 +140,7 @@ ecosystem/indexer-grpc/indexer-transaction-generator/*.yaml
 genesis_antithesis_*/
 
 # ignore cursor rules
-.cursor/rules
\ No newline at end of file
+.cursor/rules
+
+#ignore proptest regressions
```

Comment:
why do we ignore regressions? we should fix and remove the test case?

Reply:
I'd guess he meant we fix the failure and remove those files instead of ignoring? @msmouse

Reply:
I'm fine ignoring them. maybe
```
**/proptest_regressions/*
```